### PR TITLE
Bug 1812554: manifests: add openshift/oauth-proxy:v4.4 image

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: manifests/08-openshift-imagestreams.yaml [hunk 1]
```diff
@@ -84,3 +84,17 @@ spec:
...
         name: quay.io/openshift/origin-must-gather:v4.0
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  namespace: openshift
+  name: oauth-proxy
+spec:
+  tags:
+    - name: v4.4
```
Comment (sttts) [inline manifests/08-openshift-imagestreams.yaml]: put a comment why this is 4.4?

Comment (mfojtik) [reply manifests/08-openshift-imagestreams.yaml]: updated.

**Pull request merged**